subsys/bluetooth/crypto/bt_crypto.c

```diff
@@ -17,7 +21,33 @@
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(bt_crypto);
 
+#if defined(CONFIG_BT_USE_PSA_API)
+int bt_crypto_aes_cmac(const uint8_t *key, const uint8_t *in, size_t len, uint8_t *out)
+{
+	mbedtls_svc_key_id_t key_id;
```

Comment:
Why `mbedtls_svc_key_id_t` is used instead of `psa_key_id_t`?

Reply:
Ops, sorry, I worked in Mbed TLS scope for too long so this came up automatically :D. I will fix this